### ISSUE TITLE
Prevent/hide map features from rendering

### ISSUE THREAD

**Issue by AliFlux** [mapbox/mapbox-gl-native#11477]
Hi.

Is there a way to prevent map features (roads, buildings, etc...) from rendering? I am using `node-mapbox-gl-native` and there isn't much of a documentation available except for this structure:

```ruby
{
    zoom: {zoom}, // number, defaults to 0
    width: {width}, // number (px), defaults to 512
    height: {height}, // number (px), defaults to 512
    center: [{longitude}, {latitude}], // array of numbers (coordinates), defaults to [0,0]
    bearing: {bearing}, // number (in degrees, counter-clockwise from north), defaults to 0
    pitch: {pitch}, // number (in degrees, arcing towards the horizon), defaults to 0
    classes: {classes} // array of strings
}
```

I suspect that we can do this via **classes: {classes}**, however I'm not so sure about what to pass in this parameter. Are there any examples for this?

Thanks.

**Comment by asheemmamoowala** [comment]
Feature rendering is controlled by styles([full specification](https://www.mapbox.com/mapbox-gl-js/style-spec/)), which are loaded using `map.load()`.

This repo is for reporting bugs or requesting features. For help with “how do I” questions like this one, please ask on [StackOverflow](https://stackoverflow.com/questions/tagged/mapbox+ios) or reach out to our [support](https://www.mapbox.com/contact/support/) team.

**Comment by AliFlux** [comment]
Hi.

The reason why I posted this here is because the documentation is incomplete, which by all means is considered a bug.
**classes: {classes} // array of strings** is just bad documentation. What exactly is `classes`? The documentation should be updated to describe this option.

**Comment by asheemmamoowala** [comment]
@AliFlux Thanks for reporting this. `classes` is left over from paint classes which were removed in #8953. 

Ticketed out a fix in #11495.
